### PR TITLE
New innovation and delivery header

### PR DESCRIPTION
**Pull request by joaquimadraz**
### Motivation

[Trello card ](https://trello.com/c/YmMO8tXX/95-what-we-do-innovation-delivery)
Add new innovation and delivery header.

### Test plan

- Navigate to the "what we do"
- New header should show up

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/pages/what-we-do/header-slice/style.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  margin-bottom: 20px;
+}
+
+.subtitle {
+  composes: boldSansSerif from "../../../css/typography/_fonts.css";
+  font-weight: bold;
+  text-align: center;
+  line-height: 1.25;
+  margin-top: 20px;
+  font-size: 16px;
```
Comment (matt-paul) [inline site/pages/what-we-do/header-slice/style.css]: wondering if you could compose from a preexisting class within fonts?

File: site/pages/what-we-do/header-slice/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styles from './style.css';
+import arrowImg from './arrow.png';
+
+const headerSlice = () => {
+  return (
+    <div className={styles.pageHeaderContainer}>
```
Comment (matt-paul) [inline site/pages/what-we-do/header-slice/index.js]: perhaps a <section> would be marginally better semantically / for accessibility?

Comment (joaquimadraz) [reply site/pages/what-we-do/header-slice/index.js]: Make sense 👍 It also keeps the consistency with the other slices.

File: site/pages/what-we-do/header-slice/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import arrowImg from './arrow.png';
+
+const headerSlice = () => {
+  return (
+    <section className={styles.pageHeaderContainer}>
+      <div className={styles.left}>
+        <div className={styles.flexible}>
+          <h1 className={styles.pageHeader}>
+            <p>Do the</p>
+            <p>right thing</p>
```
Comment (dpiatek) [inline site/pages/what-we-do/header-slice/index.js]: This is invalid HTML. `p` tags can only be used where flow content is expected. For an `h1` only phrasing content can be used https://html.spec.whatwg.org/dev/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements

File: site/pages/what-we-do/header-slice/index.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          </h1>
+          <img role="presentation" className={styles.arrowDown} src={arrowImg} />
```
Comment (dpiatek) [inline site/pages/what-we-do/header-slice/index.js]: Using `role` here is good but having `alt=""` is actually better supported https://www.w3.org/WAI/tutorials/images/decorative/

File: site/pages/what-we-do/header-slice/index.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <h3 className={styles.pageSubHeader}>Innovation</h3>
+      </div>
+      <div className={styles.right}>
+        <div className={styles.flexible}>
+          <img role="presentation" className={styles.arrowUp} src={arrowImg} />
```
Comment (dpiatek) [inline site/pages/what-we-do/header-slice/index.js]: Same comment as above

File: site/pages/what-we-do/header-slice/index.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+          <h1 className={styles.pageHeader}>
+            <p>Do the</p>
+            <p>thing right</p>
+          </h1>
```
Comment (dpiatek) [inline site/pages/what-we-do/header-slice/index.js]: As above

File: site/pages/what-we-do/header-slice/style.css [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+.pageHeaderContainer .left {
+  text-align: left;
+  margin-right: auto;
+}
+
+.pageHeaderContainer .right {
+  text-align: right;
+  margin-left: auto;
+}
```
Comment (dpiatek) [inline site/pages/what-we-do/header-slice/style.css]: Don't think the descendant selector is needed here (ie. it should be specific enough)?

File: site/pages/what-we-do/header-slice/index.js (hunk 3 above)
Comment (joaquimadraz) [reply site/pages/what-we-do/header-slice/index.js]: Thanks for the resource!

**Comment by joaquimadraz** [comment]
@dpiatek did a couple more corrections.

**Comment by dpiatek** [comment]
:+1:

**Comment by chris-brookes** [comment]
@joaquimadraz bugs:
1. Desktop - bottom sentence font size incorrect. It is 30px, it should be 32px.
2. Alt text missing from the arrows.

screen shot 2018-03-08 at 15 32 47

File: site/pages/what-we-do/header-slice/style.css [hunk 8]
```diff
@@ -112,9 +116,15 @@
 
   .arrowUp {
     margin-right: 60px;
+    transform: rotate(-165deg)
+  }
+
+  .arrowDown {
+    transform: rotate(2deg);
```
Comment (dpiatek) [inline site/pages/what-we-do/header-slice/style.css]: Screenshots would be great to explain this change - should we not just replace the asset if we need to do this?

File: site/pages/what-we-do/header-slice/index.js [hunk 9]
```diff
@@ -10,13 +10,21 @@ const headerSlice = () => {
...
+            src={arrowImg}
+          />
         </div>
         <h3 className={styles.pageSubHeader}>Innovation</h3>
       </div>
       <div className={styles.right}>
         <div className={styles.flexible}>
-          <img alt="" className={styles.arrowUp} src={arrowImg} />
+          <img
+            alt="Arrow back up from delivery phrase to innovation phrase."
```
Comment (dpiatek) [inline site/pages/what-we-do/header-slice/index.js]: Can we check this alt text with @stevecottle ?

**Comment by chris-brookes** [comment]
[What we do: Innovation + Delivery](https://trello.com/c/YmMO8tXX/95-what-we-do-innovation-delivery)

File: site/pages/what-we-do/header-slice/index.js (hunk 9 above)
Comment (chris-brookes) [reply site/pages/what-we-do/header-slice/index.js]: @dpiatek That's already on our list :-). @joaquimadraz and I spoke about it yesterday.

File: site/pages/what-we-do/header-slice/style.css (hunk 8 above)
Comment (joaquimadraz) [reply site/pages/what-we-do/header-slice/style.css]: The arrow pointing down has a different inclination on the small screen. 
screen shot 2018-03-09 at 10 37 12
screen shot 2018-03-09 at 10 37 01

And it's the same asset for the arrow pointing up so I went for the transformation.

Comment (dpiatek) [reply site/pages/what-we-do/header-slice/style.css]: That makes sense 👍

Comment (joaquimadraz) [reply site/pages/what-we-do/header-slice/style.css]: Anyway, now that I looked at it, some unnecessary rotations are being done. I'll clean it up.

File: site/pages/what-we-do/header-slice/index.js (hunk 9 above)
Comment (dpiatek) [reply site/pages/what-we-do/header-slice/index.js]: Ok cool

Comment (stevecottle) [reply site/pages/what-we-do/header-slice/index.js]: @joaquimadraz @dpiatek What are your thoughts on tagging this up as a whole diagram? For example we put a box around the diagram and have a tag which reads something like 'Cyclic diagram of doing the right thing through innovation and delivery' and ignores the individual components. I'm thinking tagging up the individual components might not be that helpful for those who use screen readers. Also to get a screen reader to focus on each component in a circular fashion might not be possible.

Comment (joaquimadraz) [reply site/pages/what-we-do/header-slice/index.js]: @stevecottle 
That makes sense to me. It also reduce the number keystrokes to traverse the page which is good.
I still have to add the attributes for keyboard navigation.

Comment (stevecottle) [reply site/pages/what-we-do/header-slice/index.js]: @joaquimadraz Great, let's give it a go. Do you need anything from me?

Comment (dpiatek) [reply site/pages/what-we-do/header-slice/index.js]: I think that might be possible with ARIA label attributes, but maybe a simpler solution would to just have the elements as an ordered list? and then the arrow images can again have `alt=""` as order of things can be followed from HTML?

**Pull request merged**